tests/Unit/PointwiseFunctions/AnalyticData/NewtonianEuler/Test_KhInstability.cpp

```diff
@@ -93,6 +92,14 @@ void test_analytic_data(const DataType& used_for_size) noexcept {
   test_move_semantics(std::move(kh_inst_to_move), kh_inst);  //  NOLINT
 
   test_serialization(kh_inst);
+
+  // run post-serialized state through checks with random numbers
+  pypp::check_with_random_values<
+      1, typename KhInstabilityProxy<Dim>::template variables_tags<DataType>>(
+      &KhInstabilityProxy<Dim>::template primitive_variables<DataType>, kh_inst,
```

Comment:
Sorry, I meant `serailize_and_deserialize(kh_inst)` to make sure the serialization works correctly.

Reply:
Done, and I'm running it through pypp as well.

Reply:
Sorry, I still wasn't clear enough :) I meant:
```cpp
pypp::check_with_random_values<
      1, typename KhInstabilityProxy<Dim>::template variables_tags<DataType>>(
      &KhInstabilityProxy<Dim>::template primitive_variables<DataType>, serialize_and_deserialize(kh_inst),
      "KhInstability",
      {"mass_density", "velocity", "specific_internal_energy", "pressure"},
      {{{0.0, 1.0}}}, members, used_for_size);
```
so that the deserialized version is checked. I should've been clearer initially, my apologies!

Reply:
Youre good. I thought the function mutated the argument instead of returning a new one.